<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveConnection</class>
 <widget class="QDialog" name="MoveConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Curr">
     <property name="title">
      <string>Текущие подключения</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QLabel" name="label_CurShops">
          <property name="text">
           <string>Магазин</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_CurClient">
          <property name="text">
           <string>Клиент</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="Cur_Connect_Sity">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="Cur_Connect_Client">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTableWidget" name="Cur_Connect_Shop">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_CurSity">
          <property name="text">
           <string>Город</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QTableWidget" name="Cur_Connections">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Is_PC</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connection</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Is_Angry</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_CurConnections">
          <property name="text">
           <string>Текущие подключения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Dest">
     <property name="title">
      <string>Магазин для переноса</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLabel" name="label_DestShops">
          <property name="text">
           <string>Магазин</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_DestClients">
          <property name="text">
           <string>Клиент</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="Dest_Connect_Sity">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="Dest_Connect_Client">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTableWidget" name="Dest_Connect_Shop">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_DestSity">
          <property name="text">
           <string>Город</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_DestConnections">
          <property name="text">
           <string>Текущие подключения</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QTableWidget" name="Dest_Connections">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Is_PC</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connection</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Is_Angry</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_Move">
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
